izpack-core/src/main/resources/com/izforge/izpack/bin/langpacks/installer/deu.xml

```diff
@@ -122,6 +122,7 @@
     <str id="PDFLicencePanel.headline" txt="Lizenzvereinbarungen"/>
     <str id="ProcessPanel.headline" txt="Externe Prozesse ausführen"/>
     <str id="ShortcutPanel.headline" txt="Verknüpfungen"/>
+    <str id="SelectPrinterPanel.headline" txt="Select Printer"/>
```

Comment:
German translation is: "Drucker auswählen"

Reply:
Changed